### PR TITLE
Add basic unit-tests for unicode.js

### PR DESCRIPTION
**Pull request by Snuffleupagus**
Re: issue #7261.

### CODE REVIEW

File: test/unit/unicode_spec.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    });
+
+    it('should get normalized Unicode values', function () {
+      // fi => f + i
+      expect(NormalizedUnicodes['\uFB01']).toEqual('fi');
+      // Arabic
+      expect(NormalizedUnicodes['\u0675']).toEqual('\u0627\u0674');
+    });
+
+    it('should not get normalized Unicode values', function () {
```
Comment (timvandermeij) [inline test/unit/unicode_spec.js]: You have "should get normalized Unicode values" above, so shouldn't this read something like "should not get **un**normalized Unicode values"? It confuses me a bit that you have a "should" and "should not" case for the same thing you test.

File: test/unit/unicode_spec.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+  describe('getUnicodeRangeFor', function () {
+    it('should get correct Unicode range', function () {
+      // A (Basic Latin)
+      expect(getUnicodeRangeFor(0x0041)).toEqual(0);
+      // fi (Alphabetic Presentation Forms)
+      expect(getUnicodeRangeFor(0xFB01)).toEqual(62);
+    });
+
+    it('should not get a valid Unicode range', function () {
```
Comment (timvandermeij) [inline test/unit/unicode_spec.js]: Did you mean "an invalid Unicode range"?

Comment (Snuffleupagus) [inline test/unit/unicode_spec.js]: > Did you mean "an invalid Unicode range"?

Not really, since it'd be an error if we actually got a Unicode range here. Would the following clear up possible confusion: `it('should not get a Unicode range', function () {`

File: test/unit/unicode_spec.js (hunk 1 above)
Comment (Snuffleupagus) [inline test/unit/unicode_spec.js]: Would the following renaming, for the these two tests, be better?
`should get normalized Unicode values for ligatures`
and
`should not normalize standard characters`

Comment (timvandermeij) [inline test/unit/unicode_spec.js]: Yes, that sounds much better!

File: test/unit/unicode_spec.js (hunk 2 above)
Comment (timvandermeij) [inline test/unit/unicode_spec.js]: Yes, that sounds better to me.

**Comment by Snuffleupagus** [comment]
Comments have been addressed.

/botio unittest

**Comment by timvandermeij** [comment]
Looks good, thank you!
